### PR TITLE
Add toString Definition for WordArray and WordArray return type for Hash

### PR DESCRIPTION
**Pull request by hiro5id**
Please fill in this template.

[√ ] Use a meaningful title for the pull request. Include the name of the package modified.
[√ ] Test the change in your own code. (Compile and run.)
[√ ] Follow the advice from the readme.
[√ ] Avoid common mistakes.
[√ ] Run npm run lint package-name (or tsc if no tslint.json is present).
Select one of these and delete the others:

If changing an existing definition:

[√] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/brix/crypto-js/blob/develop/src/core.js#L199
[√] Increase the version number in the header if appropriate.
[√] If you are making substantial changes, consider adding a tslint.json containing { "extends": "dtslint/dt.json" }.

This pull request replaces pull request #18979

### CODE REVIEW

File: types/crypto-js/index.d.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ declare namespace CryptoJS {
 		salt: string;
 		ciphertext: string;
 		key?: string;
+        toString(encoder?: Encoder): string;
```
Comment (misak113) [inline types/crypto-js/index.d.ts]: Next time please try to keep same indention in file. However it's not merge breaker. I accept it now.

**Review by misak113 — APPROVED** [review]

**Comment by misak113** [comment]
:+1:

File: types/crypto-js/index.d.ts (hunk 1 above)
Comment (hiro5id) [inline types/crypto-js/index.d.ts]: My apologies.   WebStorm showed it all aligned nicely on my screen.  I'll investigate and learn why the difference in GitHub.  Thanks for accepting it.

Comment (hiro5id) [inline types/crypto-js/index.d.ts]: My apologies @misak113  .   WebStorm showed it all aligned nicely on my screen.  I'll investigate and learn why the difference in GitHub.  Thanks for accepting it.

**Pull request merged**